certbot/certbot/_internal/plugins/manual.py

```diff
@@ -134,9 +133,11 @@ def _verify_ip_logging_ok(self):
             else:
                 raise errors.PluginError('Must agree to IP logging to proceed')
 
-    def _perform_achall_with_script(self, achall):
+    def _perform_achall_with_script(self, achall, achalls):
         env = dict(CERTBOT_DOMAIN=achall.domain,
-                   CERTBOT_VALIDATION=achall.validation(achall.account_key))
+                   CERTBOT_VALIDATION=achall.validation(achall.account_key),
+                   CERTBOT_ALL_DOMAINS=','.join([one_achall.domain for one_achall in achalls]),
```

Comment:
Python does not really require the `[]` here because `join()` can act on a generator, not just a list. It would work equally well to say

`.join(one_achall.domain for one_achall in achalls)`

since at least Python 2.6 and probably longer ago. :slightly_smiling_face:

Reply:
Thanks, I fixed that.